csm/conf/prepare.py

```diff
@@ -115,7 +116,9 @@ def _prepare_and_validate_confstore_keys(self):
                 const.OPENLDAP_SERVERS:f"{const.OPENLDAP_SERVERS_KEY}",
                 const.OPENLDAP_ROOT_ADMIN:f"{const.OPENLDAP_ROOT_ADMIN_KEY}",
                 const.OPENLDAP_ROOT_SECRET:f"{const.OPENLDAP_ROOT_SECRET_KEY}",
-                const.OPENLDAP_BASEDN:f"{const.OPENLDAP_BASEDN_KEY}"
+                const.OPENLDAP_BASEDN:f"{const.OPENLDAP_BASEDN_KEY}",
+                const.METRICS_PERF_STATS_MSG_TYPE : const.METRICS_PERF_STATS_MSG_TYPE_KEY,
+                const.METRICS_PERF_STATS_RETENTION_PERIOD:const.METRICS_PERF_STATS_RETENTION_PERIOD_KEY
```

Comment:
Change retention period to retention size

Reply:
Modifications done